src/main/java/com/rcv/ContestConfig.java

```diff
@@ -41,6 +41,7 @@
   static final boolean DEFAULT_EXHAUST_ON_DUPLICATE_CANDIDATES = false;
   static final boolean DEFAULT_CONTINUE_UNTIL_TWO_CANDIDATES_REMAIN = false;
   static final boolean DEFAULT_TREAT_BLANK_AS_UNDECLARED_WRITE_IN = false;
+  static final boolean DEFAULT_INTEGRAL_WINNING_THRESHOLD = true;
```

Comment:
I like that this defaults to `true`, but the problem is that when the option isn't defined in a config file (e.g. if you load one of the old ones), it actually defaults to false I think because of the way `RawContestConfig` is initialized. I mentioned this in the TODO at the top of this section and brought this up in a discussion we had before:
`// TODO: if any booleans are unspecified in config file, they default to false no matter what`

What I'd suggest is you actually invert this boolean and make it `NON_INTEGER_WINNING_THRESHOLD` and have it default to `false` here. That way, it's consistent with all our other booleans defaulting to false and also, if you load up an old config file, it'll properly default to the desired value.

Reply:
Yeah, I thought about this. I really don't like the idea that we can't have any boolean variables that default to true... but I guess there's no better alternative for now.